docs/releases/4.2.txt

```diff
@@ -163,7 +163,9 @@ Migrations
 Models
 ~~~~~~
 
-* ...
+* Added the :meth:`QuerySet.comment()
+  <django.db.models.query.QuerySet.comment()>` method to allow injecting
```

Comment:
```suggestion
* The new :meth:`.QuerySet.comment` method allows ...
```

Reply:
> Do we need to do this recursively?  I think we may be more restrictive and raise an error when message contains \*, */ or --.

That's a good idea, better to let users strip or escape the strings as needed.